packages/typescript/test/function-declaration.test.tsx

```diff
@@ -76,7 +76,10 @@ it("supports type parameters by descriptor object", () => {
   const decl = (
     <FunctionDeclaration
       name="foo"
-      typeParameters={{ a: "any", b: "any" }}
+      typeParameters={[
+        { name: "a", type: "any" },
```

Comment:
I think this should be `extends: "any"` instead of type?

Reply:
Right you are.